app/install.sh

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
 
-# Copies the parameters dist file to the actual parameters.yml.
-# If you run composer manually, you will benefit from Incenteev
-# ParameterHandler, which will interactively ask you for
-# the parameters to be copied.
-if [ ! -e app/config/parameters.yml ]; then
-    cp app/config/parameters.dist.yml app/config/parameters.yml
-fi
+# Initialization script for the Bamboo web application
 
-# Firing up composer. Better to invoke the INSTALL than an UPDATE
-HOME=$(pwd) sh -c 'composer install --no-interaction'
+getopts :fh FLAG
+case $FLAG in
+    f) # force composer install
+
+       # Copies the parameters dist file to the actual parameters.yml.
+       # If you run composer manually, you will benefit from Incenteev
+       # ParameterHandler, which will interactively ask you for
+       # the parameters to be copied.
+       if [ ! -e app/config/parameters.yml ]; then
+           cp app/config/parameters.dist.yml app/config/parameters.yml
+       fi
+
+       # Firing up composer. Better to invoke the INSTALL than an UPDATE
+       HOME=$(pwd) sh -c 'composer install --no-interaction'
```

Comment:
This one runs composer in current directory, bypassing composer cache configuration, thus creating a `.composer` directory in current directory and always downloading all the packages from packagist. Is that what we want?